src/Pages/Store/Home/Component/Footer/Footer.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Footer.scss";
+
+class Footer extends Component {
+  render() {
+    return (
+      <div className="background">
```

Comment:
컴포넌트의 가장 바깥 태그의 클래스이름은 컴포넌트 이름과 동일하게 해주세요.
```suggestion
      <div className="Footer">
```

Reply:
아 넵 알겠습니다 ㅎㅎ